xrdp/Makefile.am

```diff
@@ -110,3 +110,9 @@ dist_xrdppkgdata_DATA = \
   sans-18.fv1 \
   cursor0.cur \
   cursor1.cur
+
+if MACOS
+# must be tab below. Common out Xorg and Xvnc since they don't work on MacOs. 
+install-data-hook:
+	sed -i '' '235,249s/^/;/g' $(DESTDIR)$(sysconfdir)/xrdp/xrdp.ini
```

Comment:
this breaks if the file changes in potentially minor ways

Reply:
I can bullet proof it by using grep to look for the [Xorg] and [Xvnc] blocks to get the line numbers.